src/main/java/hudson/plugins/octopusdeploy/OctopusDeployReleaseRecorder.java

```diff
@@ -305,7 +306,16 @@ public boolean perform(AbstractBuild build, Launcher launcher, BuildListener lis
             Result result = launchOcto(launcher, commands, masks, envVars, listener);
             success = result.equals(Result.SUCCESS);
             if (success) {
-                //build.addAction(new BuildInfoSummary(BuildInfoSummary.OctopusDeployEventType.Release, serverUrl + urlSuffix));
+                String serverUrl = getOctopusDeployServer(serverId).getUrl();
+                if (serverUrl.endsWith("/")) {
+                    serverUrl = serverUrl.substring(0, serverUrl.length() - 1);
+                }
+                OctopusApi api = getOctopusDeployServer(serverId).getApi().forSpace(spaceId);
+                Project fullProject = api.getProjectsApi().getProjectByName(project, true);
+                String urlSuffix = api.getReleasesApi().getPortalUrlForRelease(fullProject.getId(), releaseVersion);
+                String portalUrl = serverUrl + urlSuffix;
+                log.info("Release created: \n\t" + portalUrl);
```

Comment:
There's 1 case missing, the `Create Release` step has a `Deploy this release after it is created?` option, that also performs a deployment.

I think here you could check that flag, and also call into `getPortalUrlForDeployment`

Reply:
In 1.9.0 we dont seem to show the deployment link as well as the release link. 
But I've added it in, it will now show both links if we Create Release + Deploy